### ISSUE TITLE
PeptideShaker data import fails - BUG perhaps?

### ISSUE THREAD

**Issue by marcisaksson** [compomics/peptide-shaker#334]
[PeptideShaker Search GUI Report 2018-10-01 11.16.10.zip](https://github.com/compomics/peptide-shaker/files/2433816/PeptideShaker.Search.GUI.Report.2018-10-01.11.16.10.zip)

Hello, 

I cannot import my searchGUI results into peptideShaker  version 1.16.29. See attached zipped report files.  Previously I had some memory issues with Java. I changed the memory settings so that Java can utilize 20 GB (I have 32 RAM in total) and I use 10 threads (I have 10 cores -> 20 threads). It seems like PeptideShaker always stops at : validating identifications at 1% FDR, quality control of matches. 

Once I managed to get past this step but then the program stopped in the following step  writing;

_An error occurred: Object Marc_P1809_039.mgf_cus_Marc_P1809_039.103054.103054.3 corresponding to entry marc_test_test_0_id_ccs_"PeptideShaker|0_Marc_P1809_039.mgf_psm_parameters"_ccs_Marc_P1809_039.mgf_cus_Marc_P1809_039.103054.103054.3 not found in cache when saving._

Would be grateful if you could helpme  solve this problem!

Marc #332

**Comment by hbarsnes** [comment]
Hi Marc,

I'm afraid this is a known issue with the way PeptideShaker handles the memory and that sometimes can occur. The good news is that it should be fixed in the new major release of PeptideShaker. But there will still be some time until the new version is ready.

Besides increasing the memory another option is to load the results from less search engines at the same time. For example, try loading only the X! Tandem results. And in your case you can exclude the Comet results without the risk of loosing anything anyway, as there seems to be an error with loading the Comet pep.xml files?

Best regards,
Harald

**Comment by marcisaksson** [comment]
Harald, thanks for the quick reply!

Happy to hear that a new release might handle the issues I am having :)

You advice me to only load results from one search engine at a time. Given that I have imported for example X!Tandem results into PeptideShaker, would it be possible to append search results from another search engine, like OMSSA, later into the same project? That could be a way out for me in wait for a new release. 

Best,

Marc

**Comment by hbarsnes** [comment]
Hi Marc,

> Given that I have imported for example X!Tandem results into PeptideShaker, would it be possible to append search results from another search engine, like OMSSA, later into the same project?

No, I'm afraid not. The only way to get data from multiple search engines into the same PeptideShaker project is to load them at the same time. The gains with using multiple search engines can often be minimal though, and in our testing X! Tandem usually provides the best results. Dropping the other search engines should therefore not dramatically alter your results. But if you want to confirm this, I'd recommend loading a smaller subset of your data in PeptideShaker and checking the Spectrum IDs tab. There you will see the number of uniquely identified PSMs per search engine.

Best regards,
Harald

**Comment by marcisaksson** [comment]
Hi Harald,

I searched my data with X!Tandem only but it did not make it through unfortunately. Please see attached report/error log file. 

[hs_err_pid1992.log](https://github.com/compomics/peptide-shaker/files/2437074/hs_err_pid1992.log)

[SearchGUI Report 2018-10-02 02.28.03.zip](https://github.com/compomics/peptide-shaker/files/2437079/SearchGUI.Report.2018-10-02.02.28.03.zip)

Best,

Marc

**Comment by hbarsnes** [comment]
Hi Marc,

Seems like you managed to crash Java. Not sure why though. Maybe you can try reducing the number of threads used? In many cases it could also help simply reloading the data in PeptideShaker. Note that this can be done without having to redo the search in SearchGUI.

Best regards,
Harald

**Comment by marcisaksson** [comment]
Hi again Harald, 

I reduced the number of threads to 5 but kept the Java memory settings at 20 GB. It actually went through (thumbs up!) only with X!Tandem but I was surprised to see that only 150 proteins/groups were found (there should be > 5300 protein groups according to the latest MaxQuant version). One difference was that I had to regenerate the .mgf spectral files prior to running PeptideShaker since these were removed after an earlier failed run. I will investigate further if it could be the reason for the low ID rate. 

best,

Marc
